src/components/Checkbox/Checkbox.spec.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import {Checkbox} from './index';
+
+describe('Checkbox', () => {
+    it('should display additional class names', () => {
+        const className = 'test';
+        render(<Checkbox aria-label="checkbox" className={className}/>);
+        // Checking the parent element because additional classes are applied to the
+        // wrapping div whereas custom attributes are applied to the inner input
+        // element
+        expect(screen.getByRole('checkbox').parentElement).toHaveClass(className);
+    });
+
+    it('should add additional attributes', () => {
+        const customAttribute = 'test';
+        render(<Checkbox aria-label="checkbox" data-custom={customAttribute}/>);
```

Comment:
Why do you add `aria-label="checkbox"` ? If it's a good practice to add maybe we should add it to storybook examples?

Reply:
For some reason, I was only getting a warning when running tests without the `aria-label`:
```
console.warn
If you do not provide children, you must specify an aria-label for accessibility
```
I wasn't getting any errors when using the component the same way in the stories..going to read more into aria-labels to figure out why this warning is being thrown.